html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
```

Comment:
In this case, you are exiting the current directory just to go right back into that same directory, which is unnecessary. You can just state the name of the file since this file is in the same directory as the file you want to link to.